src/crawlee/beautifulsoup_crawler/__init__.py

```diff
@@ -1,6 +1,7 @@
 try:
-    from ._beautifulsoup_crawler import BeautifulSoupCrawler, BeautifulSoupParser
+    from ._beautifulsoup_crawler import BeautifulSoupCrawler
     from ._beautifulsoup_crawling_context import BeautifulSoupCrawlingContext
+    from ._beautifulsoup_parser import BeautifulSoupParser
```

Comment:
I would rename the `BeautifulSoupParser` to `BeautifulSoupParserType` or something like that.

And the `BeautifulSoupContentParser` to `BeautifulSoupParser`, so that it is aligned with the `Parcel`.